src/polygonParts/models/interfaces.ts

```diff
@@ -48,7 +48,7 @@ export type FindPolygonPartsOptionsFilterGeometries = Polygon | MultiPolygon;
 export type FindPolygonPartsResponse<ShouldClip extends boolean = boolean> = FeatureCollection<
   Polygon,
   ReplaceValuesOfType<
-    NonNullableRecordValues<
+    OptionalToNullableRecordValues<
```

Comment:
type named change is kinda weird,
old one was `NonNullableRecordValues` means those value cannot be null
new one is `OptionalToNullableRecordValues` means the are represented now as nullable which is kinda confusing